src/renderer/components/MintInfoView.tsx

```diff
@@ -1,13 +1,14 @@
 import * as sol from '@solana/web3.js';
 
 import Accordion from 'react-bootstrap/esm/Accordion';
-import { Button } from 'react-bootstrap';
+import { Button, Modal } from 'react-bootstrap';
 import { toast } from 'react-toastify';
 import {
   useConnection,
   useWallet,
   WalletContextState,
 } from '@solana/wallet-adapter-react';
+import React, { useState } from 'react';
```

Comment:
Where are you using `React` in here?

Reply:
its what vscode keeps telling is needed...